src/Module/Admin/Storage.php

```diff
@@ -129,6 +129,7 @@ protected function content(array $request = []): string
 				'prefix' => $storage_form_prefix,
 				'form'   => $storage_form,
 				'active' => $current_storage_backend instanceof ICanWriteToStorage && $name === $current_storage_backend::getName(),
+				'set_disabled' => DI::config()->isSetDisabled('storage', 'name'),
```

Comment:
This should be set only once in the base template variable array lines 139-151, not in each storage backend array.
```suggestion
				'is_writable' => DI::config()->isWritable('storage', 'name'),
```